posthog/api/team.py

```diff
@@ -176,10 +176,21 @@ class TeamRevenueAnalyticsConfigSerializer(serializers.ModelSerializer):
     events = serializers.JSONField(required=False)
     goals = serializers.JSONField(required=False)
     filter_test_accounts = serializers.BooleanField(required=False)
+    persons_join_mode = serializers.ChoiceField(
```

Comment:
It might be more appropriate to have this as a HogQLQueryModifier option, so that changing it will cache-bust properly

Reply:
The naming of this is very confusing. I immediately think of "persons on events", not "revenue analytics". Please make it more verbose.

I'm also not a fan of dumping everything on the team model, but 🤷